cluster-up/cluster/images.sh

```diff
@@ -6,8 +6,8 @@ declare -A IMAGES
 IMAGES[gocli]="gocli@sha256:220f55f6b1bcb3975d535948d335bd0e6b6297149a3eba1a4c14cad9ac80f80d"
 if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
     IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0@sha256:aebf67b8b1b499c721f4d98a7ab9542c680553a14cbc144d1fa701fe611f3c0d"
-    IMAGES[k8s-1.18]="k8s-1.18@sha256:f4ea6411610a4f5eb51c43d27177d7450e32240c253883a58fee102c2becffdc"
-    IMAGES[k8s-1.17]="k8s-1.17@sha256:2bb37d65749439d86e53258fb3151697d8339b9e6109aa8d23f577f123380d49"
+    IMAGES[k8s-1.18]="k8s-1.18@sha256:9728f65ab0f4e99657dae528478014aafe3c4a0a232a02e6146c8e699298861d"
```

Comment:
I should know where do you get this from, but I don't... Would you help me find the source, so I can double-check this?

Reply:
Would have been good if you had asked this yesterday; it's somewhere on prow, but looks like it's no longer available; I don't know why the reporter doesn't feature the output of both the `/test push-k8s-1.17` jobs.

I'm now looking into [where the push is done](https://hub.docker.com/r/kubevirtci/k8s-1.17/tags) and don't see those images, since there are other jobs that require newer images (https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/373/push-k8s-1.17/1266099932285112320 and https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/373/push-k8s-1.18/1266099932264140800) .

@dhiller looks like we only keep a single image on dockerhub; does this mean I'll have to wait for the other PR to get merged, rebase, generate new images, and update the `sha` ?

Reply:
Thanks to @qinqon I was able to become a little less prow illiterate; the [PR's job history](https://prow.apps.ovirt.org/?pull=368) shows them; you're looking for [1.17](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/368/push-k8s-1.17/1265914829206458368#1:build-log.txt%3A2071) and [1.18](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/368/push-k8s-1.18/1265914704484634624#1:build-log.txt%3A2086) images shas.